models/Drone.model.js

```diff
@@ -1 +1,13 @@
-// Iteration #1
\ No newline at end of file
+// Iteration #1
+const { Schema, model } = require('mongoose');
+
+const droneSchema = new Schema ({
+    name: String,
```

Comment:
Try to add [some validations](https://mongoosejs.com/docs/validation.html) to the Model fields.